apps/api/src/modules/job/job.service.ts

```diff
@@ -257,7 +257,8 @@ export class JobService {
         "You don't have permission to update this job",
       )
     }
-
+    if (job.status !== JobStatus.OPEN)
+      throw new BadRequestException("You can't update job that is not open.")
```

Comment:
Should this be `ForbiddenException` instead ?

Reply:
was thinking about it but decided to go with bad request at that moment, will change.